frontend/src/libs/components/tow-truck-card/tow-truck-card.tsx

```diff
@@ -10,19 +11,27 @@ import styles from './styles.module.scss';
 type Properties = {
   truck: TruckEntity;
   distance?: number;
-  hasFooter?: boolean;
+  hasButton?: boolean;
+  onOrderButtonClick?: () => void;
 };
 
 const TowTruckCard: React.FC<Properties> = ({
   truck,
   distance,
-  hasFooter = true,
+  hasButton = true,
+  onOrderButtonClick,
 }: Properties) => {
-  const { manufacturer, capacity, pricePerKm, towType } = truck;
+  const {
+    manufacturer: manufacturerRaw,
+    capacity,
+    pricePerKm,
+    towType,
+  } = truck;
   const img = getTowTruckImage(towType);
+  const manufacturer = manufacturerRaw.replaceAll('_', ' ');
```

Comment:
as I understood u are trying to make a readable value from its key. In such cases usually there is a map on fe, called manufacturerKeyToReadable.map, where for each manuf key we write which value we want to display